src/util/Ip.ts

```diff
@@ -60,4 +60,18 @@ export function onSameNetwork(ip1: string, ip2: string, mask: string) {
             if ((ip1Array[i] & mask) !== (ip2Array[i] & mask)) return false;
         }
     }
+    return false
```

Comment:
Which case can even end up here? Woulr be only a case with invlaid /strange input data or= Should we maybe better have a "throw new Error("invalid data provided for check")" (maybe with a better error message)

Reply:
not sure.  I didn't use the onSameNetwork or modify.